packages/design-system/src/styles/components/_Button.scss

```diff
@@ -13,40 +13,46 @@
   font-weight: var(--fontWeight, $button__font-weight);
   line-height: 1.3;
   text-align: center;
-  text-decoration-color: transparent;
   display: inline-block;
   padding: var(--space, $spacer-1 $spacer-3);
 
+  &:is(a) {
+    text-decoration: none;
+
+    &:focus {
```

Comment:
I needed to override some styles that are being applied directly to the anchor tag, so there's some duplicate CSS happening in this block. It makes me sad, but I had to do it.